### ISSUE TITLE
Issue #1 - Techniques vs occupations

### ISSUE THREAD

**Issue by chin-rcip** [chin-rcip/collections-model#1]
Philippe Michon (2019.06.21):

In AiC, we will find techniques attached to the makers like: "Person A is doing watercolour technique". Heather suggested to transform these techniques in occupations. For instance, "Watercolour" could become "Watercolorist". This is probably not a big deal to manage if the values are clean enough. That said, I'm not sure if it's exactly the same thing. If I did some artworks using watercolour technique, am I necessarily a watercolorist? I'm not sure. Let me know your thoughts.

Also, Stephen, Heather and I think that we should use a type statement on the person for this information e21->p2->e55->e55 (Techniques or Occupations). I was wondering if we should use e39 instead of e21 here. I think a group can also have a technique or an occupation.

Stephen Hart (2019.06.21):

A group can definitely have a technique, as an painters atelier may use different techniques of painting but not others

Heather Dunn (2019.06.21)

In AIC, it seems to be the technique that the artist is known for.  And the values are very clean, they come from a short bilingual controlled vocabulary.  So it should be possible to transform them to occupations if we decide that's the best approach.

Karine Leonard-Brouillet (2019.06.24):

Regarding the technique to occupation shift, I think this might become very tricky. In the case of makers that use a single technique it can be unproblematic (e.g. paint->painter), but in the case of more esoteric techniques it would introduce occupations that I do not think most artists would identify with. For example, what would Gina Pane be when considered in the context of Unanaestheticized Climb? We could translate Perfmormance->Performer, but then this would obscure the distinction between performance art (artistic performance considered in the context of contemporary art) and the performing arts(theater, dance, etc.).It would be problematic for all productions that are immaterial in part or in nature (would a conceptual artist be an ideator? what about multimedia artists or artists doing online art?). I am thinking of artists such as Richard Long (A Line Made By Walkinghttps://www.tate.org.uk/art/artworks/long-a-line-made-by-walking-ar00142) or Marina  Abramović (The House with the Ocean View https://en.wikipedia.org/wiki/Endurance_art) for example. I am also not sure why it would be more relevant to have the occupation than the medium or technique? Would it be to qualify craftsmanship such (in the case of woodworking for example)? Or is it because we are investigating how to retrieve data for the occupation field?

Philippe Michon (2019.06.28):

I agree with Karine, I would argue that we should stick to the data. I think the technique and the occupation are different and complementary fields. Contemporary art always gives perfect use cases :clin_d'œil:

@Stephen Hart to be sure:
Technique would be:
e39->p2->e55->p2->e55(Technique)
However, do we use the same pattern for the occupations with a e55(Occupation)? or we go with something like:
e39->p14i->e7->p2->e55->p2->e55(Occupation)
In other words, the same pattern as flourished dates.

Stephen Hart (2019.06.28):

I would go with the longer event path, as it will follow the same pattern as the profession (and also flourished dates, as you mention). I don’t think it complicated too much the model.
I see the technique field as a list of technique the actor used in his life.

Philippe Michon (2019.06.28)

I'm not sure if I follow you:
Technique: Simple p2 pattern
Occupation: Long path with e7
Flourished dates: Long path with e7
Am I right?

Stephen Hart (2019.06.28):

Yes.

**Comment by chin-rcip** [comment]
This has been implemented in the v1.5 of the TM

**Comment by illip** [comment]
I reopened this issue to review the pattern for the technique(s) used by an Actor for next Target Model version.

So currently, we have:

![Technique_used](https://user-images.githubusercontent.com/48293227/80635778-5f477700-8a2a-11ea-9146-3c241b9fbcea.png)

This is semantically wrong because the Actor is not a Technique; he uses a technique. Ideally, we would identify the technique in a specific production of an object. However, since we will often have a technique associated with an Actor (e.g. during his life, Jean Paul Riopelle used the technique "painting") we need a more generic pattern.

I think this is something we should model with `CRMsoc:Phase` or `F51_Pursuit` (that will reside in CRMsoc in a near future, under `CRMsoc:Phase` perhaps?) and P32_used_general_technique. Obviously, this `F51_Pursuit` doesn't have to be related to the Occupation one.

We could also use a `E13_Attribute_Assignment` strategy, but I would prefer the previous option.

What do you think?

**Comment by Christian-McCord** [comment]
**McCord Museum Use Case:** 
Nous considérons les occupations et les techniques comme des données distinctes, de même que le rôle joué par un _Actor_ dans la production d'un objet.  EX: William Notman.  Dans sa fiche _Actor_ **Occupation = Photographer**

Cette fiche est reliée à 43 433 fiches _Object_ pour lesquelles Wm Notman joue différents rôles, et parfois plus d'un rôle à la fois.  Un exercice de normalisation est ici requis mais les Roles listés se rapprochent, je crois, de votre conception d'une _Occupation_:
![image](https://user-images.githubusercontent.com/67706836/117187927-73932200-adaa-11eb-86ed-9516f8004db2.png)

Pour chacune de ces fiches _Object_ une ou des techniques de production, basées sur le AAT, sont précisées.  Wm Notman n'est pas nécessairement l'exécutant de cette technique, la fiche _Object_ peut être reliée à plus d'un _Actor_, mais il est impliqué et son implication est définie avec le champ Role.  Dans ce tableau il y a aussi place à de la normalisation (ex: certaines techniques sont en fait des noms d'objet).  Ces différentes techniques découlent de l'occupation de l'_Actor_:
![image](https://user-images.githubusercontent.com/67706836/117190336-4b58f280-adad-11eb-8713-af3e4fd02a68.png)

Requête MSAccess pour le 1er tableau:
![Capture-Notman-Roles](https://user-images.githubusercontent.com/67706836/117190540-8f4bf780-adad-11eb-8743-c94fdfc07176.JPG)

Requête MSAccess pour le 2e tableau:
![Capture-Notman-Techniques](https://user-images.githubusercontent.com/67706836/117190782-d76b1a00-adad-11eb-88d3-e8340f06ad89.JPG)

**Comment by emchateau** [comment]
Pour la qualification des artistes par technique, la facette du AAT 
http://www.getty.edu/vow/AATFullDisplay?find=painter&logic=AND&note=&subjectid=300386777

**Comment by marielmat** [comment]
In preparation for the 11th meeting of the Semantics Committee (June 10)

Q : Is the technique associated with the objects or actors, or both? | What pattern is best for both options?

A : When the technique/role are **associated with the objects, the option 1 pattern** seem the most appropriate. 
When **associated with the actors**, I don't understand how a Pursuit can used a technique or have a type instead of using P14.1 in the role of?

In our database, the role of the artist is specified in each object record. For the production of object _x_, the artist/maker played the role of _y_. The techniques are also associated with the objects.
<img src="https://user-images.githubusercontent.com/61242683/120653497-ba7a4300-c44e-11eb-9753-ada58f8aec56.png" width=60% height=60%>

For the techniques, we follow the Info-Muse rule for "[Techniques de fabrication](https://www.musees.qc.ca/fr/professionnel/guidesel/doccoll/fr/ethno-art-techno/tf.htm)". 
They are more often participative adjectives ("adjectif participe", pas certaine de la traduction) declined in order of importance. The information recorded appears to me to be much more about different stages of transformation of a material before it became a manufactured object. 
For a tapestry, we might have "Woven; Embroidered" (Tissé; Brodé) while for a piece of furniture it might look like "Carved; Turned; Painted; Nailed" (Sculpté; Tourné; Peint; Cloué). 
Also, the Info-Muse rule says that we can use "Handcrafted" and "Industrial". We don't use them at the MCQ anymore (because the line between the two is not always clear), but we still have several entries with these terms.

**Comment by stephenhart8** [comment]
The semantic committee reviewed this issue during its last meeting, which was held on 2021/06/10. No decision has been made yet, but here are some items to consider:

1. Techniques used during the production of objects should be modelled using the property `P32_used_general_technique` and an instance of `E55_Type`. It should be linked to the instance of `E12_Production` representing the production event of the object. It has been accepted by the Semantic Committee.
2. It is still uncertain how to model a technique used by actors, but not related to an object (for instance mentioned in a biography).
   1. The option proposed was to link the technique used to an instance of `F51_Pursuit` modelling the occupation event of an actor. In this occupation, the actor could use several techniques. The issue with this option is that it requires creating a fictive `F51_Pursuit`, which could lead to induce false results or too much noise during queries.
   2. The first pattern proposed was to model the techniques used as instances of `E55_Type` and link it directly to the actor through the property `P2_has_type`. But as stated here in this Github issue, semantically this would mean that an actor is of a type "oil painting", which is not the case. However, the CRM SIG has stated that it is still possible to employ such a pattern, in [issue #429](http://www.cidoc-crm.org/Issue/ID-429-p72-has-language) about language spoken by actors. It would then be possible to still use the `P2_has_type` `E55_Type` pattern to describe techniques. If needed, an instance of `E17_Type_Assignment` can be modelled to state who has assigned this type, if the data provenance pattern is not sufficient.
   3. Another option, similar to the `P2_has_type` `E55_Type` is to link the technique (modelled as an instance of `E55_Type`) to the actor through a new property that could be called "composed of knowledge / have skill". In this way, we would avoid the semantic inaccuracy of the `P2_has_type` `E55_Type` option. However, this would mean creating a new property, which is something CHIN has avoided up to now.
3. It has been stated that it would be useful to type actors in certain categories, with the purpose of classifying and organizing them. This classification process, vastly used in the heritage sector, and especially in museums, provides a lot of information on how actors are seen in those heritage institutions. Therefore, even if this categorization process is flawed (as it only represents one vision of a multifacet actor), it could be beneficial to model it as documentation on the reception of artists and other producers of artefacts by contemporary institutions, as long as data provenance is also recorded. This should probably be the subject of a new issue, as it may also be applied to objects.

**Comment by stephenhart8** [comment]
To illustrate those options, here is an example of the production of the Mona Lisa (La Joconde), with techniques used in the production of the painting (Peinture à l'huile), and some other techniques linked directly to the actor (Peinture à l'huile, Fresque and Sculpture sur marbre)

| Object     | Object technique   | Actor            | Actor Technique                                   |
| ---------- | ------------------ | ---------------- | ------------------------------------------------- |
| La Joconde | Peinture à l'huile | Léonard de Vinci | Peinture à l'huile, fresque, sculpture sur marbre |

![Issue#1_Example](https://user-images.githubusercontent.com/49342191/122048587-c1576d00-cde1-11eb-8c09-64d3143de90d.png)

**Comment by Christian-McCord** [comment]
Voici un exemple d'une œuvre composite (photographie et peinture combinées) qui a nécessité l'intervention de 2 Actors:
-Wm. Notman & Son (studio de photographie)
-Eugène L'Africain (peintre)
Dans leurs fiches Actor respectives, **Wm. Norman & Son** est une "Photographic firm" (champ Job Title) et **Eugène L'Africain** est un "Artist" (occupation).  Dans la fiche Object, les techniques suivantes sont précisées: 
![image](https://user-images.githubusercontent.com/67706836/122079914-daedba00-cdcb-11eb-95d0-8649a15961c6.png)
C'est le "Role" de l'Actor qui précise la technique qui lui est associée:
![image](https://user-images.githubusercontent.com/67706836/122080432-43d53200-cdcc-11eb-8b29-7cbde6115e5d.png)
Fiche sommaire de l'Object:
![image](https://user-images.githubusercontent.com/67706836/122081193-e55c8380-cdcc-11eb-8933-869c6c22b4c8.png)

**Comment by stephenhart8** [comment]
The semantic committee reviewed this issue during its last meeting, which was held on 2021/07/15. It has been decided:

- The documentation of technique is documented in the production event of objects.
- Techniques directly associated with actors should be considered as practice or skills, but the pattern should be named around the idea of technique. It would be a categorization, rather than a description of a biographic event. It would therefore be documented with an instance of E55_Type through the property P2_has_type. A metatype is also needed on top of this instance of E55_Type , and should be called “technique used”.
- It is important to document the difference between an occupation, which is the documentation of a dated biographic event in the life of the actor, and this technique used type, which is a categorization made by an institution on an actor.
- It is also important to indicate in the documentation that the technique used pattern cannot be dated.

Once implemented in the Target Model 2.2., this issue can be closed.

However, a few questions, yet unsolved, must be answered in the future, in other issues:

- Issue #74: Is it necessary to add an E13_Attribute_Assignment in the technique used pattern?
- Issue #75: How to link a technique, documented in the production of an object, to the actor involved in the production?
- Issue #76: What vocabulary should be used to distinguish:
  - the technique used in a production of an object
  - the technique used linked to an actor
  - the occupation as a biographical event?
